### PR TITLE
docs: add ngIndia to event page

### PR DESCRIPTION
**Pull request by santoshyadavdev**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Comment by santoshyadav198613** [comment]
Cc: @kapunahelewong @StephenFluin

File: aio/content/marketing/events.html [hunk 1]
```diff
@@ -143,6 +126,27 @@ <h1 class="banner-headline no-toc no-anchor">Events</h1>
...
+        </th>
+        <td>Rome, Italy</td>
+        <td>Oct 6th workshops, 7th conference, 2019</td>
+      </tr>
+      <!-- ReactiveConf 2019 -->
+      <tr>
+        <th><a href="https://reactiveconf.com/" title="ReactiveConf">ReactiveConf</a></th>
+        <td>Prague, Czech Republic</td>
+        <td>October 30 - November 1, 2019</td>
+      </tr>
```
Comment (kapunahelewong) [inline aio/content/marketing/events.html]: Thanks for moving these past events into the next section, too. As they are listed from the most recent at the top, could you move them to this location? https://github.com/angular/angular/blob/master/aio/content/marketing/events.html#L50

They should be in reverse order from what they currently are, so:
1. ReactiveConf
2. NgRome
3. Angular Connect 
4. NG-DE and so on as it currently is listed.

**Review by kapunahelewong — CHANGES_REQUESTED** [review]

File: aio/content/marketing/events.html (hunk 1 above)
Comment (santoshyadav198613) [reply aio/content/marketing/events.html]: Done.

**Review by kapunahelewong — APPROVED** [review]
Thanks, @santoshyadav198613 😊

**Review by StephenFluin — APPROVED** [review]

**Pull request closed without merging**